scripts/enum/skill.lua

```diff
@@ -68,3 +68,63 @@ xi.skill =
     -- 60~63 unused
     -- MAX_SKILLTYPE = 64
 }
+
+xi.subskill =
+{
+    XBOW_SHORTBOW = 0,
+    GUN           = 1,
+    CANNON        = 2,
+    SHURIKEN      = 3,
```

Comment:
I think boomerangs and chakrams also have the same "sweet spot" as shuriken in the distance calcs SE uses. We can probably just rename this "thrown"